confgenerator/confgenerator_test.go

```diff
@@ -148,15 +166,19 @@ func updateOrCompareGolden(t *testing.T, testName string, expected string, actua
 }
 
 func TestGenerateConfigsWithInvalidInput(t *testing.T) {
-	filePath := invalidTestdataDir
+	filePath := invalidTestdataDir +  "/linux"
+	hostInfo, _ := host.Info()
```

Comment:
It looks like the `init` method at line 52 determines the value for `isWindows`. Could we reuse that here?